ui/lib/core/addon/components/confirm-action.js

```diff
@@ -33,8 +31,6 @@ export default Component.extend({
   confirmMessage: 'You will not be able to recover it later.',
   confirmButtonText: 'Delete',
   cancelButtonText: 'Cancel',
-  disabledTitle: "Can't delete this yet",
```

Comment:
Did these get removed?

Reply:
Yes, because when the button is disabled this message won't come up anyway